README.md

```diff
@@ -1,22 +1,103 @@
 # Celeste on iOS
 
-## what you'll need
-- an **FNA** build of Celeste (this repo has been tested with the Steam version only, the itch.io version *should* work too)
-- MacOS (if you don't have access to a MacOS machine, you can use a MacOS VM)
-- XCode
-- Visual Studio for Mac with the C#, .NET and Xamarin.iOS workloads
-- patience
-
-## how to build
-- install the needed software mentioned above
-- run `./build.sh`
-- follow the instructions
-- hope it works :3
-
-these instructions might be slightly wrong or have missing information because i wrote this like 3 months ago and don't remember every detail - if you have issues feel free to open an issue :3
-
-## installing
-the script will tell you where the final .ipa file is located - just sideload it to your iOS device using something like TrollStore, AltStore or similar
+# How to Build
+
+This guide will walk you through building a Celeste IPA for iOS using macOS. Please note that **only macOS** is supported for this process. If you don't have a Mac you'll have to use a Virtual Machine.
+
+## Prerequisites
+
+1. **Install Xcode**  
+   - Make sure you have Xcode installed and set up correctly. You can download it from the [Mac App Store](https://apps.apple.com/us/app/xcode/id497799835).
+   - Install the iOS Simulator in Xcode
+
+2. **Install Git**  
+   If Git isn't already installed, running `git` from your terminal will often prompt to install it.
+
+3. **Install FMOD Engine**  
+   - Go to the FMOD download page: [FMOD v1.10.09](https://www.fmod.com/download?version=1.10.09#modengine).  
+   - You will need to create a free FMOD account to download.
+   - Make sure that the FMOD version is set to **v1.10.09 (unsupported)**
+   - Download the **iOS** version of **FMOD Engine**
+   - After downloading, mount the downloaded `fmodstudioapi11009ios-installer.dmg` file by double-clicking it. **No need to do anything further**; simply ensure it's mounted.
+
+4. **Install Visual Studio for macOS**  
+   - Download Visual Studio for macOS from [here](https://visualstudio.microsoft.com/vs/older-downloads/).
+     - If that download link doesn't work, use [this one](https://visual-studio.macupdate.com)
+   - Follow the websites links until it lets you download **Visual Studio 2022 for Mac**
+   - Follow the installation instructions.
+   - When prompted during installation, make sure to select the following components:
+     - `.NET`
+     - `.Net Multi-Platform App UI`
+     - `Android`
```

Comment:
i am very sure that you do *not* need the android workload to build an iOS app c:

Reply:
This is true, but during the installation, when you select .Net Multi-Platform App UI, both android and iOS are automatically selected, and if you deselect Android (which i think people might) it automatically deselects .Net Multi-Platform App UI, so I thought it was best to leave in android to avoid confusion, But again happy to take it out